croissant-rdf/pyproject.toml

```diff
@@ -0,0 +1,119 @@
+[project]
+name = "croissant-rdf"
+version = "0.1.2"
+description = "Tools for working with RDF from Croissant JSON-LD resources."
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [{ name = "David Steinberg", email = "david@resium.com" }]
+maintainers = [{ name = "David Steinberg", email = "david@resium.com" }]
+keywords = [
+    "CroissantML",
+    "RDF",
+    "JSON-LD",
+    "SPARQL",
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+]
+
+requires-python = ">=3.8"
+dependencies = [
+    "rdflib >=6.0.0",
```

Comment:
The dependencies are almost the same as mlcroissant. Why not include this library in mlcroissant?

You'd also benefit of the CI/CD that is already in place.

Reply:
I think that would be great but is a bigger lift! Maybe we could consider that follow on work or if you or someone else wanted to carry it over the line, what do you think?